testdata/apps/spin-app.yaml

```
@@ -16,7 +16,7 @@ spec:
       runtimeClassName: wasmtime-spin-v2
       containers:
       - name: spin-hello
-        image: ghcr.io/spinkube/spin-operator/hello-world:20240909-145818-g30def7d
+        image: ghcr.io/spinframework/spin-operator/hello-world:20240909-145818-g30def7d
```

```suggestion
        image: ghcr.io/spinframework/spin-operator/hello-world:20250206-205102-gf007281
```

(ref https://github.com/spinframework/runtime-class-manager/pull/339#issuecomment-2695824220)